README.md

```diff
@@ -516,6 +518,56 @@ hermione_base_url=http://example.com hermione path/to/mytest.js
 hermione_browsers_firefox_sessions_per_browser=7 hermione path/to/mytest.js
 ```
 
+## Programmatic API
+
+To use Hermione in your own scripts or build tools you can use it experimental programmatic API.
+First you need to create and configure new Hermione instance:
+
+```javascript
+var Hermione = require('hermione/lib/hermione');
+
+var hermione = new Hermione({
+    specs: [
+        ...
+    ],
+    browsers: {
+        ...
+    }
+});
+```
+
+* `new Hermione(filePath)` will load configuration from file at given path;
+
+* `new Hermione(config)` will create configuration from passed object (see configuration
+  file reference). Options **must** have `specs` and `browsers` settings specified.
+
+* `new Hermione(config, options)` extend configuration after checks with some additional properties from options.
+
+* `new Hermione(config, options, {cli: true, env: true})` allow configuration properties overriding with values passed from env or cli.
+
+### Accessing the config options
+
+You can get values of options via `hermione.config` property:
+
+```javascript
+var Hermione = require('hermione/lib/hermione');
+    hermione = new Hermione('/path/to/config');
+
+console.log(hermione.config.baseUrl);
+
+```
+
+### Running tests
+
+Use `hermione.run(paths, browsers)` method.
+
+* `paths` - array of file paths or directories to run the tests.
+
+* `browsers` — array of browser ids to execute tests in. By default, tests are
+  executed in all browsers, specified in configuration.
+
+Returns promise which will be resolved with flag of successful tests execution or rejected if critical error occurred.
```

Comment:
Returns promise which will be resolve with `true` or `false` in dependency of successful tests execution or rejected... ?